src/processor/common.ts

```diff
@@ -0,0 +1,16 @@
+import { ExportableAssignmentRule, ExportableSdRule } from '../exportable';
+
+// Places general Quantity-setting rules ahead of Quantity.unit setting rules
+export function switchQuantityRules(rules: ExportableSdRule[]): void {
+  const seenRules = new Map();
+  rules.forEach((rule, index) => {
+    if (!seenRules.has(rule.path) && rule instanceof ExportableAssignmentRule)
+      seenRules.set(rule.path, index);
+    const unitRulePath = rule.path.concat('.unit');
```

Comment:
I think it is fair to assume that `Quantity.unit` is the only usage of `unit` in FHIR right? I think I'm good with assuming that.

Reply:
Actually now that I think about it, I'm not worried about this at all, because worst case we would just be re-ordering the rules in a way that I can't picture being harmful.

Reply:
I did a search of [the FHIR datatypes page](https://www.hl7.org/fhir/datatypes.html) and that _seems_ to be the case, but as you said there shouldn't be any adverse effects if that's not the case.